src/lib/cache.svelte.ts

```
@@ -0,0 +1,213 @@
+
+export async function addFoodToCache(food: number) {
+	// Foods
+	const {data: foodsData, error: foodsError} = await supabase
+		.from("foods")
+		.select("id, name, by_volume, calories, protein")
+		.eq("id", food)
+	if (foodsError) { console.error("Error in getting food:", foodsError); toast.error("Error in getting food."); return }
+	foodsData.forEach(food => foods[food.id] = {
+		...food,
+		servings: {},
+	})
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)